docs/api.rst

```diff
@@ -460,6 +460,25 @@ All objects from ``attrs.validators`` are also available from ``attr.validators`
       x = field(validator=attrs.validators.and_(v1, v2, v3))
       x = field(validator=[v1, v2, v3])
 
+.. autofunction:: attrs.validators.or_
+
+   For example:
+
+   .. doctest::
+
+      >>> from typing import Union
+      >>> @define
+      ... class C:
+      ...     val: Union[int, str] = field(validator=attrs.validators.or_(attrs.validators.instance_of(int), attrs.validators.instance_of(str)))
```

Comment:
Yep, I was aware of the fact you can can `isinstance` with type argument being a tuple, but didn't put 2 and 2 together that it will be the case with `attrs.validators.instance_of` as well. 🤦🏻‍♂️

Reply:
Done in a0b6b241a05b825077decccdf99af43eab35ab34 .

I reworked the example to have `one-or-many` scenario, where the attribute can either a value of a type or a collection of values of the same type, encountered eg. in parsing the query parameters in HTTP URIs.

```html
...?val=1&val=2&val=3
```

Is this OK?

---

As an aside, `docs` seem to be tested wth py312, so I don't have to do the `typing` ceremony and can write `val: int | list[int]`, but since there are python versions in the test matrix that don't have this syntax on by default, should I keep it as it is now or rework to conform to newer versions?

Thanks!

Reply:
Yes, please use the most-modern syntax. Let’s teach ppl modern features.

Example looks fine on a phone 😅